packages/engine/src/ecs/classes/World.ts

```diff
@@ -19,16 +19,13 @@ export interface ECSWorld extends IWorld {
 let worldIds = 0
 export class World {
   static worlds: Map<number, World> = new Map<number, World>()
-  static defaultWorld: World
+  static defaultWorld: World = new World()
   ecsWorld: ECSWorld
   entities: Entity[]
   portalEntities: Entity[]
   pipelines: EnginePipelines
 
   constructor() {
-    if (typeof World.defaultWorld === 'undefined') {
```

Comment:
This seems to be resolving a deeper ECS issue we have, if world isn't starting before we are calling this